pkg/trace/writer/sender.go

```diff
@@ -347,6 +359,7 @@ var ppool = &sync.Pool{
 		return &payload{
 			body:    &bytes.Buffer{},
 			headers: make(map[string]string),
+			retries: 0,
```

Comment:
When something is at its zero-value (in this case `0` for `int32`), it doesn't need to be specified in the struct definition. You can remove this line.